backend/src/store/room.go

```diff
@@ -182,6 +187,13 @@ func (r *Room) AutoRevealIfCan() {
 	}
 }
 
+func (r *Room) AutoRevealIfTimeIsUp() {
+	if r.autoRevealTimer > 0 {
```

Comment:
Go has a nice functionality around timers:
https://gobyexample.com/timers
eg.: we can stop and resets them 

This implementation will be error-prone because anytime you set a new timer a new sleep will be created. For example changing the timer to a bigger value will not clear the timer that is already running and the cards will be revealed early then you would expect. Also if you set the timer to lower value the old, bigger one will still run and can overflow to the next voting session.